### ISSUE TITLE
can't use SET after 0.1.3 and any further versions - lost session

### ISSUE THREAD

**Issue by skaradub** [analytikaplus/clickhouse-tableau-connector-jdbc#26]
hello there!

I had bug in some instances where I couldn't disable session_id to be generated. So I made an issue. You fixed it, but in the version fixing and all the other after it I can't use SET initial SQL, driver looses session_id.. 
So session id is not set, but initial SQL can't be used at all (even with set_session_id=1)

I have 2020.3 version.

regards,
Yury

![image](https://user-images.githubusercontent.com/57046561/151040822-a3048542-b43e-4539-9b8e-1599402f6902.png)

**Comment by artshevchenko** [comment]
Hi! What version of connector and driver are you using?

**Comment by skaradub** [comment]
> Hi! What version of connector and driver are you using?

Hi! 

all the others after 0.1.2 =) the error is the same

driver is the one you have in instructions:
0.1* - 3.1
0.2 - 3.2

**Comment by skaradub** [comment]
No, I lied, 0.2* have diffferent error, all the above is about 0.1.3 and 0.1.4

**Comment by yurifal** [comment]
Could you please describe what parameters are you trying to configure? 
Some of them may not be strictly session-bound, and they could be set query-wise.
Please try to add them in the Advanced --> URL Parameters first.

**Comment by artshevchenko** [comment]
Hi! Probably not worth focusing on version 0.1.*. What is the error text in 0.2.* with the 0.3.2-patch1/2 driver?

**Comment by skaradub** [comment]
Hello everyone! 
We made some research with 0.3.2 and found out that new driver itself changed body and headers and our cache server doesn't like its new behaviour (we have smth like balancer with cache before servers)
When we connected new driver to one of the servers directly it worked. 
So for now we try to find changes in JDBC and then rewrite our cache server, problem is not with the connector itself.

Agree with you that 0.1.* is useless time spending for the future needs.
